data_model/src/query/mod.rs

```diff
@@ -970,18 +962,39 @@ pub mod peer {
         #[display(fmt = "Find all peers")]
         #[ffi_type]
         pub struct FindAllPeers;
+    }
+
+    /// The prelude re-exports most commonly used traits, structs and macros from this crate.
+    pub mod prelude {
+        pub use super::FindAllPeers;
+    }
+}
+
+pub mod executor {
+    //! Queries related to [`crate::executor`].
+
+    #[cfg(not(feature = "std"))]
+    use alloc::{format, string::String, vec::Vec};
+
+    use derive_more::Display;
+
+    queries! {
+        /// [`FindExecutorDataModel`] Iroha Query finds the data model of the current executor.
+        #[derive(Copy, Display)]
+        #[display(fmt = "Find executor data model")]
+        #[ffi_type]
+        pub struct FindExecutorDataModel;
 
-        /// [`FindAllParameters`] Iroha Query finds all [`Peer`]s parameters.
+        /// [`FindAllParameters`] Iroha Query finds all defined executor configuration parameters.
         #[derive(Copy, Display)]
         #[display(fmt = "Find all peers parameters")]
-        // TODO: Unused query. Remove?
         #[ffi_type]
         pub struct FindAllParameters;
```

Comment:
```suggestion
        pub struct FindAllParameters;
```

so what does this query do now? It finds values of parameters?

Reply:
if yes, it shouldn't be in the `executor` module because it also returns values for core parameters

Reply:
Yes, it returns all currently set executor parameters. In this PR, I haven't yet introduced the concept of core parameters.

As we discussed previously (if memory doesn't fail me), after proper core params implementation, we are going to have a single composite query `FindParameters` which will return both executor's key-value map and strictly typed set of core params.